jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/CpuUtilizationHelper.java

```diff
@@ -79,7 +84,13 @@ synchronized double getSystemCpuLoad() {
 		}
 
 		if ((latestTime.getTimestamp() - oldestTime.getTimestamp()) >= MINIMUM_INTERVAL) {
-			cpuLoad = calculateCpuLoad(latestTime, oldestTime);
+		
+		    if (latestTime.getCpuLoad() == -1) {
```

Comment:
Instead of duplicating this logic, can you add it to the `calculateCpuLoad()` method?  That way all future callers will get the right behaviour without having to code this extra check themselves

Reply:
Comment addressed in  ffdf65f8be493319c906b24c60b752782364cb35